gitbook-docs/security.md

```diff
@@ -66,6 +77,7 @@ If the login is successful, the server will send a response like the following:
   "state": "COMPLETED",
   "result": 200,
   "login": {
+  	"expiry": 86400,
```

Comment:
How about `timetolive` or something along that line? Would more intuitivite to me.

Reply:
now `timeToLive`